test/docfx.RegressionTest/RegressionTest.cs

```diff
@@ -368,5 +384,27 @@ private static async Task SendAzureDevOpsPullRequestComments(int prId, string co
                 response.EnsureSuccessStatusCode();
             }
         }
+
+        private enum BuildReason
+        {
+            Commit = 0, // default, when build reason is commit build
+            PullRequest = 1, // pull request build
+            Schedule = 2, // daily scheduled build
+        }
+
+        private static BuildReason GetBuildReason()
+        {
+            var buildReasonStr = Environment.GetEnvironmentVariable("BUILD_REASON");
+
+            // local debug should be set as PullRequest build to avoid accidentally push
+            if (buildReasonStr == null)
+            {
+                return BuildReason.PullRequest;
+            }
+
+            return Enum.TryParse(buildReasonStr, true, out BuildReason buildReason)
+                ? buildReason
+                : BuildReason.Commit;
```

Comment:
![image](https://user-images.githubusercontent.com/42199316/86333826-017c1900-bc7f-11ea-8d76-df21d7822c88.png)
in server-build, still `commit` should be the default value.
for it can be triggered for many reasons.